hnn_core/tests/test_gui.py

```diff
@@ -69,7 +69,6 @@ def test_gui_upload_params():
     assert gui.drive_widgets[-1]['tstop'].value == 250.
     gui._simulate_upload_drives(file1_url)
     assert gui.connectivity_widgets[0][0].children[1].value == 0.01
-    assert gui.drive_widgets[-1]['tstop'].value == 0.
```

Comment:
This test does not apply anymore, the poisson drive (that is not used, hence tstop=0) is no longer added when `legacy_mode=False`